cicd/3-app/javabuilder/template.yml.erb

```diff
@@ -1032,8 +1031,8 @@ Resources:
         Period: 60
         EvaluationPeriods: 1
         DatapointsToAlarm: 1
-        Threshold: 10
-        ComparisonOperator: GreaterThanThreshold
+        Threshold: 1
+        ComparisonOperator: GreaterThanOrEqualToThreshold
```

Comment:
Will this re-alarm if a second classroom is blocked in the same period after the first classroom was blocked? I don't think it's very likely, but just want to make sure we don't miss unblocking the second classroom

Reply:
it won't, but it would give us a number of how many were blocked during the evaluation period so we will know if more than one classroom was blocked.